README.md

```diff
@@ -9,6 +9,7 @@ The repository contains implementations for `FlexLayout`, `HorizontalLayout` and
 
 ## Running the component demo
 Run from the command line:
+- `mvn install -DskipTests`
 - `mvn  -pl vaadin-ordered-layout-flow-demo -Pwar install jetty:run`
```

Comment:
Should the "install" from the second command then be removed? This change should be reflected to every flow component readme, right?